index.bs

```diff
@@ -206,268 +175,455 @@ interface ImageCapture {
+  <dd>A sequence of supported {{MeteringMode}} settings for <a>exposure</a> calculation.</dd>
+
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>exposureCompensation</code></dfn></dt>
+  <dd>This reflects the supported range of <a>exposure compensation</a>. The supported range can be, and usually is, centered around 0 EV.</dd>
+
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>iso</code></dfn></dt>
+  <dd>This reflects the permitted range of <a>ISO</a> values.</dd>
+
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>focusMode</code></dfn></dt>
+  <dd>A sequence of supported {{MeteringMode}} settings for <a>focus mode</a>.</dd>
+
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>brightness</code></dfn></dt>
+  <dd>This reflects the supported range of <a>brightness</a> setting of the camera. Values are numeric. Increasing values indicate increasing brightness.</dd>
+
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>contrast</code></dfn></dt>
+  <dd>This reflects the supported range of <a>contrast</a>. Values are numeric.  Increasing values indicate increasing contrast.</dd>
+
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>saturation</code></dfn></dt>
+  <dd>This reflects the permitted range of <a>saturation</a> setting. Values are numeric. Increasing values indicate increasing saturation.</dd>
+
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>sharpness</code></dfn></dt>
+  <dd>This reflects the permitted <a>sharpness</a> range of the camera. Values are numeric. Increasing values indicate increasing sharpness, and the minimum value always implies no sharpness enhancement or processing.</dd>
+
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>zoom</code></dfn></dt>
+  <dd>This reflects the <a>zoom</a> value range supported by the UA.</dd>
 
+  <dt><dfn dict-member for="MediaTrackCapabilities"><code>torch</code></dfn></dt>
+  <dd>A boolean indicating whether camera supports <a>torch</a>  mode- on meaning supported.</dd>
 </dl>
 
-# MediaSettingsRange
+## {{MediaTrackConstraintSet}} dictionary ## {#mediatrackconstraintset-section}
+
+{{MediaTrackConstraintSet}} dictionary is used for both reading the current status with {{getConstraints()}} and for applying a set of constraints with {{applyConstraints()}}.
+
+<div class="note">
+{{MediaTrackSettings}} can be retrieved to verify the effect of the application by the user agent of the requested {{MediaTrackConstraints}}. Some constraints such as, e.g. <a>zoom</a>, might not be immediately applicable.
+</div>
 
 <pre class="idl">
-  interface MediaSettingsRange {
-      readonly attribute double max;
-      readonly attribute double min;
-      readonly attribute double current;
-      readonly attribute double step;
+  partial dictionary MediaTrackConstraintSet {
+    MeteringMode       whiteBalanceMode;
+    MeteringMode       exposureMode;
+    MeteringMode       focusMode;
+    ConstrainBoolean   pointsOfInterest;
```

Comment:
@jan-ivar said:
> @miguelao Note that MediaTrackConstraintSet is effectively the dictionary passed to applyConstraints, so a boolean here leaves us no way to actually set the Point2D array.

(I got it via e-mail, but the comment seems lost in this interface :?, hence I copied it here again).

Changed to `sequence<Point2D>`;